### ISSUE TITLE
Supervisor version 234 crashes on Ubuntu 18.04.5 LTS

### ISSUE THREAD

**Issue by DrBlokmeister** [home-assistant/supervisor#1932]
**Home Assistant release with the issue:**

- Home assistant: 0.113.0
- Supervisor: 234

**Operating environment (HassOS/Generic):**

Installation on a generic Linux host

- Intel NUC Kit NUC8i5BEH
- Ubuntu 18.04.5 LTS

**Supervisor logs:**

```
20-08-16 18:38:48 INFO (SyncWorker_2) [supervisor.docker.interface] Attach to hassioaddons/grafana-amd64 with version 5.1.1
20-08-16 18:38:48 INFO (SyncWorker_1) [supervisor.docker.interface] Attach to homeassistant/amd64-addon-rpc_shutdown with version 2.2
20-08-16 18:38:48 INFO (SyncWorker_2) [supervisor.docker.interface] Attach to hassioaddons/ssh-amd64 with version 7.0.1
20-08-16 18:38:48 INFO (SyncWorker_0) [supervisor.docker.interface] Attach to hassioaddons/mqtt with version 1.2.0
20-08-16 18:38:48 INFO (SyncWorker_3) [supervisor.docker.interface] Attach to hassioaddons/sonweb-amd64 with version 0.11.0
20-08-16 18:38:48 INFO (SyncWorker_1) [supervisor.docker.interface] Attach to homeassistant/amd64-addon-deconz with version 6.2.3
20-08-16 18:38:48 INFO (SyncWorker_5) [supervisor.docker.interface] Attach to hassioaddons/influxdb-amd64 with version 3.7.3
20-08-16 18:38:48 INFO (SyncWorker_4) [supervisor.docker.interface] Attach to homeassistant/amd64-addon-mosquitto with version 5.1
20-08-16 18:38:48 INFO (SyncWorker_6) [supervisor.docker.interface] Attach to homeassistant/amd64-addon-mariadb with version 2.2.1
20-08-16 18:38:48 INFO (SyncWorker_2) [supervisor.docker.interface] Attach to hassioaddons/vscode-amd64 with version 2.4.0
20-08-16 18:38:48 INFO (SyncWorker_7) [supervisor.docker.interface] Attach to homeassistant/amd64-addon-check_config with version 3.4.0
20-08-16 18:38:48 INFO (SyncWorker_8) [supervisor.docker.interface] Attach to hassioaddons/tautulli with version 1.5.0
20-08-16 18:38:48 INFO (SyncWorker_9) [supervisor.docker.interface] Attach to hassioaddons/portainer-amd64 with version 1.2.1
20-08-16 18:38:48 INFO (SyncWorker_0) [supervisor.docker.interface] Attach to homeassistant/amd64-addon-samba with version 9.2.0
20-08-16 18:38:48 INFO (MainThread) [supervisor.snapshots] Found 18 snapshot files
20-08-16 18:38:48 INFO (MainThread) [supervisor.discovery] Load 2 messages
20-08-16 18:38:48 INFO (MainThread) [supervisor.ingress] Load 0 ingress session
20-08-16 18:38:48 INFO (MainThread) [supervisor.misc.secrets] Load Home Assistant secrets: 56
20-08-16 18:38:48 ERROR (MainThread) [supervisor.core] Detected unsupported OS: Ubuntu 18.04.5 LTS
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/docker/api/client.py", line 261, in _raise_for_status
    response.raise_for_status()
  File "/usr/local/lib/python3.8/site-packages/requests/models.py", line 941, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http+docker://localhost/v1.40/images/sha256:72ae8ee35a6664e0f5dc84d1374c4555dafa915726765502827e12236030ee03/json

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/src/supervisor/supervisor/__main__.py", line 47, in <module>
    loop.run_until_complete(coresys.core.setup())
  File "uvloop/loop.pyx", line 1456, in uvloop.loop.Loop.run_until_complete
  File "/usr/src/supervisor/supervisor/core.py", line 147, in setup
    if await self.sys_run_in_executor(self.sys_docker.check_denylist_images):
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/supervisor/supervisor/docker/__init__.py", line 240, in check_denylist_images
    for image in self.images.list():
  File "/usr/local/lib/python3.8/site-packages/docker/models/images.py", line 364, in list
    return [self.get(r["Id"]) for r in resp]
  File "/usr/local/lib/python3.8/site-packages/docker/models/images.py", line 364, in <listcomp>
    return [self.get(r["Id"]) for r in resp]
  File "/usr/local/lib/python3.8/site-packages/docker/models/images.py", line 316, in get
    return self.prepare_model(self.client.api.inspect_image(name))
  File "/usr/local/lib/python3.8/site-packages/docker/utils/decorators.py", line 19, in wrapped
    return f(self, resource_id, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/docker/api/image.py", line 245, in inspect_image
    return self._result(
  File "/usr/local/lib/python3.8/site-packages/docker/api/client.py", line 267, in _result
    self._raise_for_status(response)
  File "/usr/local/lib/python3.8/site-packages/docker/api/client.py", line 263, in _raise_for_status
    raise create_api_error_from_http_exception(e)
  File "/usr/local/lib/python3.8/site-packages/docker/errors.py", line 31, in create_api_error_from_http_exception
    raise cls(e, response=response, explanation=explanation)
docker.errors.APIError: 500 Server Error: Internal Server Error ("readlink /var/lib/docker/overlay2: invalid argument")
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

**Description of problem:**
Yesterday the home assistant supervisor panel wouldn't open but instead gave an error stating it couldn't find entrypoint.js
Reboot of the NUC did not resolve the issue
Re-installing latest version of supervisor did not resolve the issue
After performing a rollback to supervisor version 230 the issue was solved, supervisor runs correctly and I can open the supervisor panel in the webUI.

**Comment by DrBlokmeister** [comment]
Thanks @pvizeli!
Just out of curiosity, what happened and what did you do to fix it?

Can I just update to the latest docker image?

**Comment by pvizeli** [comment]
It does not fix anything. It just not stop startup if the local docker installation is mess up. which allows people to run `ha su repair` as last hope they can fix the corrupt filesystem.

**Comment by DrBlokmeister** [comment]
So you're saying that I still have an issue which is probably my fault. The update you did simply allows me to fix the issue more easily?

**Comment by pvizeli** [comment]
> probably my fault

Your system, yes. Somethings like a hard power off or others corrupted your docker filesystem database.

**Comment by DrBlokmeister** [comment]
Interesting. There was no power failure on my system. The only thing I noticed is a supervisor update, after which I had this error. I yesterday reverted to version 230 and it was fine. This morning, I again couldn't reach the supervisor, and I found out it had auto-updated to the latest version. Now I updated to the latest version, and although I still have the error, startup continues due to your fix.

So I'm wondering if it is really my issue, or an issue introduced by the latest version.
